test/e2e/operator_test.go

```diff
@@ -226,6 +212,101 @@ func TestAWSLoadBalancerControllerWithDefaultIngressClass(t *testing.T) {
 	}
 }
 
+// TestAWSLoadBalancerControllerWithCredentialsSecret tests the basic happy flow for the operator
+// using the explicitly specified credentials secret.
+func TestAWSLoadBalancerControllerWithCredentialsSecret(t *testing.T) {
+	t.Log("Creating aws load balancer controller instance with credentials secret")
+
+	// provision AWS credentials for ALBC imitating the manually created secret
+	albcSecretName := "manual-aws-credentials-e2e"
+	albcSecretRef := corev1.ObjectReference{
+		Name:      albcSecretName,
+		Namespace: "aws-load-balancer-operator",
+	}
+	albcCrName := types.NamespacedName{
+		Name:      "mimic-manual-creds",
+		Namespace: "openshift-cloud-credential-operator",
+	}
+	albcCr, err := albcCredentialsRequest(albcCrName, albcSecretRef, "testsa")
+	if err != nil {
+		t.Fatalf("failed to generate desired credentials request to mimic manually created albc credentials: %v", err)
+	}
+	if err := kubeClient.Create(context.TODO(), albcCr); err != nil {
+		t.Fatalf("failed to create credentials request to mimic manually created albc credentials: %v", err)
+	}
+	defer func() {
+		waitForDeletion(t, kubeClient, albcCr, defaultTimeout)
+	}()
+
+	name := types.NamespacedName{Name: "cluster", Namespace: "aws-load-balancer-operator"}
+	alb := newALBCBuilder().withName(name).withCredSecret(albcSecretName).build()
+	if err := kubeClient.Create(context.TODO(), alb); err != nil && !errors.IsAlreadyExists(err) {
```

Comment:
Should the test just ignore an already-exists error?  It seems to me that the test results will be unreliable if some CR that this test didn't create exists.